modules/apps/commerce/commerce-discount-service/src/main/java/com/liferay/commerce/discount/internal/target/ApplyToSKUCommerceDiscountTargetImpl.java

```diff
@@ -45,7 +45,7 @@
 		"commerce.discount.target.key=" + CommerceDiscountConstants.TARGET_SKUS,
 		"commerce.discount.target.order:Integer=20"
 	},
-	service = {CommerceDiscountSKUTarget.class, CommerceDiscountTarget.class}
+	service = CommerceDiscountTarget.class
```

Comment:
Hey @MrJohn1911, the interface `CommerceDiscountSKUTarget` seems useless. Please remove that. Thx